designer/public/js/index.js

```diff
@@ -23,6 +23,7 @@ $(function () {
     addMenuItem('Master', showMaster);
     addMenuItem('Worker', showWorkers);
     addMenuItem('Device', showDevices);
+    addMenuItem('edge', showEdge);
     addMenuItem('uService', showEndPointService);
     addMenuItem('Stream', showStreams);
```

Comment:
001Edge should be Cloud and other should be Edge1, Edge2 and so on. Correspondingly.